src/client/datascience/notebook/cellLanguageService.ts

```diff
@@ -41,14 +45,29 @@ export class NotebookCellLanguageService implements IExtensionSingleActivationSe
             (metadata?.kernelspec as IJupyterKernelSpec | undefined)?.language ||
             this.lastSavedNotebookCellLanguage;
 
-        // Default to python language only if the Python extension is installed.
-        const defaultLanguage = this.pythonExtensionChecker.isPythonExtensionInstalled ? PYTHON_LANGUAGE : 'plaintext';
         // Note, what ever language is returned here, when the user selects a kernel, the cells (of blank documents) get updated based on that kernel selection.
-        return translateKernelLanguageToMonaco(jupyterLanguage || defaultLanguage);
+        // 99% of the users today are Python, hence lets default to python,
+        // Changing the kernel will change the languages of the cells.
+        return translateKernelLanguageToMonaco(jupyterLanguage || PYTHON_LANGUAGE);
```

Comment:
Note the change (previously anyone installing Jupyter for first time will end up creating a notebook and have language set to `plaintext`.
As 99% of the users today are python users, lets default to Python.

Yes we could have typescript, r, julia users, but if you change the kernel to one of theose, then the language of the cells get updated.

This way users get syntax highlighting straight off the bat.

Also related issue is https://github.com/microsoft/vscode-jupyter/issues/5927